docs/tutorial/authentication-tutorial.md

```diff
@@ -307,12 +307,10 @@ import { getUser, isLoggedIn, logout } from "../services/auth" // highlight-line
 
 // highlight-start
 export default () => {
-  const content = { message: "", login: true }
-  if (isLoggedIn()) {
-    content.message = `Hello, ${getUser().name}`
-  } else {
-    content.message = "You are not logged in"
-  }
+  const greetingMessage = isLoggedIn()
```

Comment:
Thanks for this PR! However, I would recommend leaving the code block as it is seeing as replacing if/else syntax with ternary operators might add some cognitive load on the learner to understand what `?` `:` does. 

The above code example is easier to read. Let me know your thoughts.

Reply:
@Ekwuno, you are right, I haven't thought about that, but `content.login` is still confusing here, because it's not used anywhere else. 
Should I  update 
```
const greetingMessage = isLoggedIn()
   ? `Hello ${getUser().name}`
   : "You are not logged in"
```
into
```
let greetingMessage = '';
if (sLoggedIn()) {
   greetingMessage = `Hello ${getUser().name}`;
} else {
   greetingMessage = "You are not logged in"
}
```
?

Reply:
Oh yes! Please. I looked through and `content.login` isn't used. This suggestion is great! Update and let's get this merged!